app/api/v_sphere/virtual_machine.rb

```diff
@@ -271,10 +271,6 @@ def request
       Request.accepted.find { |each| name == each.name }
     end
 
-    def managed_folder_entry
```

Comment:
Why this change?
It will most likely break inserting VMs into folders!

Reply:
Because the method exists twice.

Reply:
Oh, that makes sense, GitHub just hid that fact from me 🤦‍♂️